test/manuscript/manuscript.test.ts

```diff
@@ -15,15 +15,14 @@
  */
 /// <reference types="@types/jest" />;
 import { schema } from '@manuscripts/transform'
-import { promises as fs } from 'fs'
+// import { promises as fs } from 'fs'
```

Comment:
why did you comment that out? if we don't need those imports then just remove them the please. This one and the one under